webrtc-stats.html

```diff
@@ -1338,6 +1339,17 @@ <h2>
                   data for that frame.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalInterFrameDelaySquared</code></dfn> of type <span class=
+                "idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Sum of the squared interframe delay in ms between consecutive frames. The
+                  interframe variance can be calculated from this value and <a>framesDecoded</a>
+                  at two timestamps <code>t0</code> and <code>t1</code>.
```

Comment:
Can you clarify that t0 and t1 are measured at the same time that framesDecoded goes up, and that the measurements are in seconds?

Reply:
Is this better?

Reply:
What you are computing is: (currentInterFrameDelay - averageInterFrameDelay)^2  or (currentInterFrameDelay)^2

Reply:
Are we trying to use this forumula? https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Na%C3%AFve_algorithm

If so...
1: Can you say which formula we are expecting to use?
2: If the missing term is (sum of interframe delays)^2, and we are estimating this based on (timestampSecond-timestampFirst)^2, but that this breaks down when you stop receiving frames - how about we add a second metric which is totalInterframeDelay? That way we have everything :))))))))

Reply:
I think we are then going for -- (totalInterFrameDelaySquared - totalInterframeDelay^2/framesDecoded)/framesDecoded

Reply:
That's a great idea to also add totalInterframeDelay, that removes all approximations on the way to get the variance and also makes the calculation more self-contained.

Reply:
Done, see the latest commit.